app/helpers/value_groups_helper.rb

```diff
@@ -1,28 +1,40 @@
 module ValueGroupsHelper
   def value_group_pie_data(value_group)
-    values = value_group.children
-      .map do |child|
-        {
-          label: to_accountable_title(Accountable.from_type(child.name)),
-          percent_of_total: child.percent_of_total.round(1).to_f,
-          value: child.sum.amount.to_f,
-          currency: child.sum.currency.iso_code,
-          currency_symbol: child.sum.currency.symbol,
-          formatted_value: format_money(child.sum.amount.to_f, precision: 2, unit: child.sum.currency.symbol, separator: ".", delimiter: ","),
-          bg_color: accountable_bg_class(child.name),
-          fill_color: accountable_fill_class(child.name)
-        }
-      end
-      .filter { |child| child[:value] > 0 }
+    values = value_group.children.map do |child|
+      {
+        label: to_accountable_title(Accountable.from_type(child.name)),
+        percent_of_total: child.percent_of_total.round(1).to_f,
+        value: child.sum.amount.to_f,
+        value_str: {
+          fractional_part: child.sum.cents_str,
+          main_part: format_money_without_symbol(child.sum.amount.to_f, precision: 0, unit: child.sum.currency.symbol)
+        },
+        currency: {
+          iso_code: child.sum.currency.iso_code,
+          symbol: child.sum.currency.symbol,
+          displayed_before_value: child.sum.currency.default_format.start_with?("%u")
```

Comment:
i have added this value to known how to display the symbol in the front like : 

![image](https://github.com/maybe-finance/maybe/assets/39769262/032feeff-ccfd-48c7-abad-9561cab13dc0)

And

![image](https://github.com/maybe-finance/maybe/assets/39769262/9b0a415e-c95e-465a-8c8c-24af3841b122)


Where sometimes it is on the front of the value and sometimes in the back